genie-agent/src/main/java/com/netflix/genie/agent/execution/services/impl/JobSetupServiceImpl.java

```diff
@@ -151,6 +151,78 @@ public File createJobDirectory(final JobSpecification jobSpecification) throws S
         return setupFiles;
     }
 
+    public Map<String, String> setupJobEnvironment(
```

Comment:
Our IDEs are still on different style settings i need to copy yours.

Reply:
ok lets look at it tomorrow